x/valaddr/types/errors.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+import (
+	"cosmossdk.io/errors"
+)
+
+var (
+	ErrInvalidIPAddress   = errors.Register(ModuleName, 1, "invalid IP address format")
+	ErrIncorrectValidator = errors.Register(ModuleName, 2, "invalid signer address")
```

Comment:
[nit] the error message and the error name don't match. Can the error or error message be renamed to include more info? For example if this is only returned when a validator does not exist for the provided signer address then perhaps


```suggestion
	ErrInvalidValidator = errors.Register(ModuleName, 2, "invalid validator")
```

and then the usage can be like

```go
return errors.Wrapf(types.ErrInvalidValidator, "signer %v does not match a validator", signer)
```